### ISSUE TITLE
Unit testing

### ISSUE THREAD

**Issue by dmarcos-au** [archit737/code-challenge#1]
https://github.com/archit737/code-challenge/blob/3eaa897237c27d1ccd6aad272f20cbbf0a70be89/src/test/java/com/transurban/utils/cloud/GoogleDriveInteractTest.java#L17

This requires a link to an account for it to successfully run. Ideally the external services would be mocked out

**Comment by archit737** [comment]
Correct! Ideally we would have mock services running on `dev` or `test` environments but in this case I just used the one service that I built.
